Source/Plugins/Core/com.equella.core/js/tsrc/search/components/SearchResult.tsx

```diff
@@ -0,0 +1,241 @@
+ * with the License. You may obtain a copy of the License at:
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import * as React from "react";
+import { SyntheticEvent } from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import { languageStrings } from "../../util/langstrings";
+import {
+  ListItem,
+  Theme,
+  Typography,
+  ListItemText,
+  Divider,
+  List,
+  ListItemIcon,
+  Collapse,
+} from "@material-ui/core";
+import { DateDisplay } from "../../components/Date";
+import {
+  InsertDriveFile,
+  ExpandMore,
+  ExpandLess,
+  AttachFile,
+  Subject,
+} from "@material-ui/icons";
+
+const useStyles = makeStyles((theme: Theme) => {
+  return {
+    inline: {
+      display: "inline",
+    },
+    heading: {
+      fontWeight: "bold",
+      paddingRight: theme.spacing(1),
+    },
+    thumbnail: {
+      paddingRight: theme.spacing(2),
+    },
+    placeholderThumbnail: {
+      paddingRight: theme.spacing(2),
+      width: "88px",
+      height: "auto",
+      color: theme.palette.text.secondary,
+    },
+    itemDescription: {
+      paddingBottom: theme.spacing(1),
+    },
+    additionalDetails: {
+      flexDirection: "row",
+      display: "flex",
+      paddingTop: theme.spacing(1),
+    },
+    status: {
+      textTransform: "capitalize",
+    },
+    nested: {
+      paddingLeft: theme.spacing(4),
+    },
+  };
+});
+
+export interface SearchResultProps {
+  /**
+   * A json representation of the SearchResult data. TODO: Properly type this when search2 endpoint is merged
+   */
+  resultData: any;
+  /**
+   * Function will be invoked when the SearchResult list item is clicked
+   */
+  onClick: () => void;
+}
+export default function SearchResult({
+  resultData,
+  onClick,
+}: SearchResultProps) {
+  const classes = useStyles();
+
+  const searchResultStrings = languageStrings.searchpage.searchresult;
+
+  const {
+    name,
+    uuid,
+    description,
+    displayFields,
+    modifiedDate,
+    status,
+    displayOptions,
+    attachments,
+  } = resultData;
+
+  const [attachExapanded, setAttachExpanded] = React.useState(
+    displayOptions.standardOpen
+  );
+
+  const handleAttachmentPanelClick = (event: SyntheticEvent) => {
+    event.stopPropagation();
+    setAttachExpanded(!attachExapanded);
+  };
+
+  const thumbnail = (disableThumb: boolean) => {
+    if (attachments.length > 0 && !disableThumb) {
+      return (
+        <>
+          <img
+            className={classes.thumbnail}
+            src={attachments[0].links.thumbnail}
+            alt={attachments[0].description}
+          />
+        </>
```

Comment:
There is only one component, a fragment wrapper isn't needed.
```suggestion
        <img
          className={classes.thumbnail}
          src={attachments[0].links.thumbnail}
          alt={attachments[0].description}
        />
```

Reply:
Yeah, i had only just discovered react fragments, so i just went a bit crazy throwing them in everywhere